### PR TITLE
Suppress feature notification

### PR DESCRIPTION
**Pull request by nilllzz**

### CODE REVIEW

File: src/app/components/grid/client.service.ts [hunk 1]
```diff
@@ -81,6 +84,13 @@ export class GridClient {
 	bootstrapReceived = false;
 	bootstrapTimestamp = 0;
 
+	/**
+	 * Store ids of posts the user has featured.
+	 * The Grid client will ignore any incoming feature notifications for posts recorded here,
+	 * because users that feature posts should not get notified about those exact posts.
+	 */
+	featuredPostIds: number[] = [];
```
Comment (YLivay) [inline src/app/components/grid/client.service.ts]: Use a `Set` for this.

Comment (YLivay) [reply src/app/components/grid/client.service.ts]: also, can't this be `private`?

Comment (nilllzz) [reply src/app/components/grid/client.service.ts]: A set? You mean a setter?
Can you tell me where we would need to access a setter for this variable?

Comment (YLivay) [reply src/app/components/grid/client.service.ts]: a capital S `Set`. as in `Set<number>`

Comment (nilllzz) [reply src/app/components/grid/client.service.ts]: Ah sure, why not.

**Pull request merged**